### ISSUE TITLE
Can not register an asset

### ISSUE THREAD

**Issue by bassjobsen** [AschPlatform/asch#221]
### Version
1.3.6 development - localnet

### Environment

Ubuntu 18.04 LTS

### Steps to reproduce
- navigate to http://localhost:4096/#/asset
- click Registered Assets
- fill in the form

### What is expected?

a new asset created (or an error)

### What is actually happening?

screen fades out. Nothing happens after that

![assets](https://user-images.githubusercontent.com/954055/43317312-a732b6e8-919c-11e8-928e-8b7f182b2304.png)

**Comment by a1300** [comment]
@bassjobsen I followed your steps and I get the following error: `You have not registered a publisher yet`. The error message is in the center of the screen. Is it possible that chrome is somehow smaller and you don't see the error message?

The normal process would be:
1. Register publisher
2. Wait 10 sec for next block
3. Register asset
4. Wait 10 sec for next block
5. Mint tokens (only available in Chinese language, you can see it on the screen, but not translated yet)

**Comment by bassjobsen** [comment]
@a1300 thanks for your quick reply, i had create a publisher already. Still can not create an asset, see also https://github.com/bassjobsen/howto-asch-docs/blob/master/create-asset.md for the steps i did

**Comment by a1300** [comment]
@bassjobsen you could try the following:

Change the directory to some random Dapp:
Execute:
```bash
asch-redeploy --asch pointToAsch --publisher pub --asset NEW
```
I would like to know if that works.

Nice tutorial by the way :+1:

**Comment by bassjobsen** [comment]
yes, with `asch-redeploy` does it works very well

**Comment by a1300** [comment]
@bassjobsen for me the registration of publisher and asset worked (via frontend)

However I had another problem: I needed to change to another __tab__ after the registration of the publisher to see that the publisher was registered.

I would try to delete the `blockchain.db` file and try again.

**Comment by bassjobsen** [comment]
okay, thanks i will try. Yes i found the same issue too.

**Comment by bassjobsen** [comment]
I have found out that this issue is UI related. The popup to confirm my action does not show up (hidden by CSS code). Most likely it also depends of your browser. The pop up does not show up for Google Chrome Version 67.0.3396.99 (Official Build) (64-bit).

**Comment by eagleHovering** [comment]
thanks , It fixed in v1.4.1 .
